stubs/AMQP.php

```diff
@@ -66,11 +66,16 @@
 
 /**
  * When publishing a message, the message must be routed to a valid queue. If it is not, an error will be returned.
+ * Note: This flag does not currently work in pecl-amqp, and WILL cause undefined behavior if you use it.
+ * @link https://bugs.php.net/bug.php?id=62786
```

Comment:
My bad. The problem is that php-amqp extension doesn't deal with `basic.return` method from broker. I slightly recall that #23 but mixed MANDATORY and IMMEDIATE from there, sorry for that. I'll add an issue (or better submit PR) for that.